File: LeetCode-35--search-insert-position.py
class Solution(object):
    def searchInsert(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        #day two: python mirror refresher by translating written java code
        for i in range (len(nums)):
            if nums[i] == target:
                return i
            
            if target < nums[i]:
                return i
        return len(nums)
        

File: LeetCode-35---search-insert-position.java
class Solution {
    public int searchInsert(int[] nums, int target) {  
        //day two: java refresher on arrays
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] == target) {
                return i;
            }

            if (target < nums[i]) {
                return i;
            }
        }
        return nums.length; //end of list reached
    }
}
